test/safe-hts-precompile/SafeHTS.js

```diff
@@ -204,16 +204,6 @@ describe('SafeHTS library Test Suite', function () {
     await utils.associateWithSigner(signer1PrivateKey, fungibleTokenAddress);
     await utils.associateWithSigner(signer1PrivateKey, nonFungibleTokenAddress);
 
-    await safeOperationsContract.safeGrantTokenKycPublic(
```

Comment:
Was wondering why these changes are here in this PR? Is it related to the local-node being bumped to 2.25.2?

Reply:
This was done due to changes related to hip 540 that were introduced in the lastet network node images. The current test scenario was not working and needed to be reworked.